src/com/itacademy/eshop/interfaces/Warnings.java

```diff
@@ -0,0 +1,6 @@
+package com.itacademy.eshop.interfaces;
+
+public interface Warnings {
+
+    public void isKidSafe();
```

Comment:
method could be called something like displaySafetyInformation() since that is what you actually doing, isKidSafe implies that you will return boolean